web/themes/new_weather_theme/templates/block/block--weathergov-hourly-forecast.html.twig

```diff
@@ -60,8 +60,8 @@
             {% for period in content.hours %}
               <td style="white-space:pre;">
                 <div class="display-flex flex-row flex-align-center">
-                  <span>{{period.windSpeed}}</span>
-                  <svg role="img" aria-hidden="true" class="width-2 height-2 margin-left-05 padding-2px" data-wind-direction="{{period.windDirection}}">
+                  <span>{{period.windSpeed}} {{ "mph" | t }}</span>
+                  <svg role="img" aria-hidden="true" class="width-2 height-2 margin-left-05 padding-2px" style="transform: rotate({{ period.windDirection.angle + 180 }}deg);">
```

Comment:
The +180 is to account for wind direction being reported as *from*, but the arrow representing *to*.